.travis.yml

```diff
@@ -1,25 +1,33 @@
 language: scala
+cache:
+  directories:
+    - $HOME/.ivy2
```

Comment:
The build is _waaaay_ faster the second time around when the cache kicks in. ~2 minutes roundtrip on Travis for me.